qiskit/algorithms/minimum_eigen_solvers/vqe.py

```diff
@@ -139,10 +138,6 @@ def __init__(self,
         if optimizer is None:
             optimizer = SLSQP()
 
-        # set the initial point to the preferred parameters of the variational form
-        if initial_point is None and hasattr(var_form, 'preferred_init_points'):
-            initial_point = var_form.preferred_init_points
-
```

Comment:
This logic needs to be retained for VQE to correctly start at the desired state as indicated from the var form and initial state combination. Pehaps we can find some better way going forwards but this is the mechanism we have now.

Reply:
Which initial state and variational form will be using this? The new `UCC` class will not have a `preferred_init_points` attribute.